docs/getting-started/local-setup.md

```diff
@@ -15,6 +15,13 @@ and validate that they are correctly configured to run on your machine.
 - **Git**
 - **`make` (for running Makefiles)**
 - **Go 19+**
+- **Check that you have in `.bashrc` or `.zshrc` :**
+
+     `export GOPATH="$HOME/go"` 
+
+     `PATH="$GOPATH/bin:$PATH"`  
+
+    **and if you don't have the commands, add this in file at the end.**
```

Comment:
````suggestion
- **Go Environment Setup**:
  - Make sure `$GOPATH/bin` is added to your `$PATH` variable.
  - To do this, you can add the following line to your `.bashrc` or `.zshrc` or other config file:
```
 export PATH=$GOPATH/bin:$PATH
```
````

IIRC, since Go 1.11, `GOPATH` is set by default during installation to` $HOME/go`. Going by this, it is sufficient to mention adding `$GOPATH/bin` to `$PATH`.

Other than that, I just worded it a bit differently :)

Reply:
Thanks for your comment but with after installing Go.1.22, I don't have `$GOPATH` automaticly confirgurate`

Reply:
I like your new worded :)

Reply:
Hmm, I am also on an M2 MBP and I checked all my .zshrc/.zprofile configs and I don't have a single line defining `$GOPATH`. I know I had an issue where my terminal and my IDE terminal were looking at different configs (I use Goland). This might be the issue, but running `echo $GOPATH` results in `$HOME/go`. 

Sure, let me change the suggestion up a little and you can commit it :)

Reply:
I use M1 and my friend who has a same problem he use M3. You didn't have a same problem it's strangh 🤨 I go look my config maybe I forget something

Thanks for you change ;) 
I validate that